### PR TITLE
[new release] conduit, conduit-mirage, conduit-lwt, conduit-lwt-unix and conduit-async (2.3.0)

### PR DESCRIPTION
**Pull request by samoht**
A network connection establishment library

- Project page: <a href="https://github.com/mirage/ocaml-conduit">https://github.com/mirage/ocaml-conduit</a>
- Documentation: <a href="https://mirage.github.io/ocaml-conduit/">https://mirage.github.io/ocaml-conduit/</a>

##### CHANGES:

* conduit-mirage: simplify the API to not mix functors and first-class
  modules anymore. We just use functors now and rely on the mirage tool
  to apply them properly (mirage/ocaml-conduit#376, @samoht)
* add client-side TLS certificate validation using OS trust anchors for
  `conduit-lwt-unix` and Mozilla's NSS for `conduit-mirage` (mirage/ocaml-conduit#375, @samoht)

### CODE REVIEW

**Comment by samoht** [comment]
(I'd like to make sure all the revdeps are fixed for this one, as I'll follow up with a corresponding cohttp release)

File: packages/conduit/conduit.2.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "Anil Madhavapeddy" "Thomas Leonard" "Thomas Gazagnaire" "Rudi Grinberg"
+]
+license: "ISC"
+tags: "org:mirage"
+homepage: "https://github.com/mirage/ocaml-conduit"
+doc: "https://mirage.github.io/ocaml-conduit/"
+bug-reports: "https://github.com/mirage/ocaml-conduit/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune"
```
Comment (mseri) [inline packages/conduit/conduit.2.3.0/opam:13]:
```suggestion
  "dune" {>= "2.0.0"}
```
according to the dune project

**Comment by mseri** [comment]
What is the state of this? Do we really need the solver help as part of this PR?

**Comment by samoht** [comment]
I've rebased my PR. I'm happy to merge it even if that solver is still confused. Hopefully everyone will switch to opam 2.1 soon :p

**Comment by samoht** [comment]
I believe this is good to go now.

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by hannesm** [comment]
FWIW I'd expect git-cohttp-mirage 3.2.0 and 3.3.0 to be constrained to conduit & conduit-mirage < 2.3.0 as well!?

**Pull request merged**